.github/workflows/trivy-scan.yml

```
@@ -0,0 +1,226 @@
+  trivy-scan:
+    name: Trivy scan (fs + image)
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.12)

[error] the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)